test/extended/testdata/build-extended/bc-scripts-in-the-image.json

```diff
@@ -0,0 +1,34 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-image"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git",
+        "ref": "s2-assemble-only"
+      }
+    },
+    "strategy": {
+      "sourceStrategy": {
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "wildfly:10.0",
+          "namespace": "openshift"
+        },
+        "runtimeImage": {
+          "kind": "DockerImage",
+          "name": "phpcoder/hello-world-runtime:0.2"
```

Comment:
We're using simple image in other cases to be sure that scenarios will fail if S2I don't use mapping from BuildConfig. If we'll use this custom image then it will be possible to have broken S2I and working tests in the same time.